components/tools/OmeroPy/test/conftest.py

```diff
@@ -96,7 +96,7 @@ def pytest_configure(config):
 
 
 try:
-    import xdist  # noqa
+    import xdist.plugin  # noqa
```

Comment:
Works for me! - need the same fix in ```OmeroWeb``` too.

Reply:
Done